airbyte-integrations/connectors/source-xero/source_xero/manifest.yaml

```diff
@@ -0,0 +1,2823 @@
+version: 0.78.5
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: CustomRecordExtractor
+      class_name: source_xero.components.CustomExtractor
+      field_path: ["{{ parameters.extractor_path }}"]
+
+  requester:
+    type: HttpRequester
+    url_base: "https://api.xero.com/api.xro/2.0/"
+    http_method: "GET"
+    authenticator:
+      type: "BearerAuthenticator"
+      api_token: "{{ config['access_token'] }}"
+    request_headers:
+      "Xero-Tenant-Id": "{{ config['tenant_id'] }}"
+      "Accept": "application/json"
+    error_handler:
+      type: CompositeErrorHandler
+      error_handlers:
+        - type: DefaultErrorHandler
+          response_filters:
+            - type: HttpResponseFilter
+              action: IGNORE
+              http_codes:
+                - 403
```

Comment:
I guess we should add 401 status code also with error message asking user to update access token and failed action.